test/index.ts

```diff
@@ -43,12 +43,23 @@ for (let envVar of Object.keys(environmentVariables)) {
     if (match) {
         let [, option] = match;
         let value: string | number = environmentVariables[envVar];
-        if (typeof value === 'string' && !isNaN(parseInt(value, undefined))) {
-            value = parseInt(value, undefined);
+
+        try {
+            value = JSON.parse(value);
+        } catch
+        {
+            if (typeof value === 'string' && !isNaN(parseInt(value, undefined))) {
+                value = parseInt(value, undefined);
+            }
         }
         options[option] = value;
     }
 }
+
+options.reporterOptions = {
+    "reporterEnabled": "mocha-junit-reporter, spec"
```

Comment:
Did you add this here just for the sake of debugging? Doesn't it overwrite what's in the env?

But actually I like this better than trying to read it from the env. I think it makes sense to read things like `MOCHA_grep` or `MOCHA_enableTimeouts` from the env since those change somewhat often, but I don't think the reporter stuff will change since you're automatically writing results to both places now. Seems simpler to just set it one place (here).

Reply:
Had it there originally.  I'm fine with that.